coverage/pom.xml

```diff
@@ -77,7 +89,13 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
-      <artifactId>livy-scala-api</artifactId>
```

Comment:
This is the build error, since currently livy-scala-api module is not existed, so it will be failed. It occurs in our in house build, but I'm not why it is not happened in travis build. @tc0312